policies.go

```diff
@@ -550,21 +551,21 @@ func (t *tokenAwareHostPolicy) getMetadataForUpdate() *clusterMeta {
 
 // resetTokenRing creates a new tokenRing.
 // It must be called with t.mu locked.
-func (m *clusterMeta) resetTokenRing(partitioner string, hosts []*HostInfo) {
+func (m *clusterMeta) resetTokenRing(partitioner string, hosts []*HostInfo) error {
```

Comment:
Now returns an error, but it's never checked in any place where we call `resetTokenRing()`

Reply:
Agree, its tech debt I need to tidy up in a future refactor.  Shouldn't affect us.